api/v2/api_test.go

```diff
@@ -37,15 +37,16 @@ const (
 )
 
 var (
-	hash           = "QmPY5iMFjNZKxRbUZZC85wXb9CFgNSyzAy1LxwL62D8VGr"
-	api            *API
-	db             *gorm.DB
-	engine         *gin.Engine
-	testRecorder   *httptest.ResponseRecorder
-	authHeader     string
-	cfg            *config.TemporalConfig
-	fakeLensClient *mocks.FakeIndexerAPIClient
-	fakeOrchClient *mocks.FakeServiceClient
+	hash             = "QmPY5iMFjNZKxRbUZZC85wXb9CFgNSyzAy1LxwL62D8VGr"
+	api              *API
+	db               *gorm.DB
+	engine           *gin.Engine
+	testRecorder     *httptest.ResponseRecorder
+	authHeader       string
+	cfg              *config.TemporalConfig
+	fakeLensClient   *mocks.FakeIndexerAPIClient
+	fakeOrchClient   *mocks.FakeServiceClient
+	fakeSignerClient *mocks.FakeSignerClient
```

Comment:
hm dont think we should do this, mocks should be created on a per-test basis because theres counting stuff that goes on under the hood

Reply:
just make sure mocks themselves are instantiated for each test case only (since you set properties on them `ReturnsOnCall` etc and check things like call counts), and are not shared